tests/codegen/enum/unreachable_enum_default_branch.rs

```diff
@@ -25,11 +25,12 @@ pub fn implicit_match(x: Int) -> bool {
     (x >= A && x <= B) || x == C
 }
 
+// Broken :(
 // The code is from https://github.com/rust-lang/rust/issues/110097.
 // We expect it to generate the same optimized code as a full match.
 // CHECK-LABEL: @if_let(
-// CHECK-NEXT:  start:
-// CHECK-NEXT: insertvalue
+// CHECK: start:
```

Comment:
A regression in a test case. :(

Reply:
I believe I wrote a bad test case. The original example is to use different values: https://github.com/rust-lang/rust/issues/110097.

Reply:
For reference: https://rust.godbolt.org/z/a3v75MdY8.